sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -557,7 +555,9 @@ def assert_counter_exists(metrics, namespace, name, step):
   # this test is flaky when state duration is low.
   # Since increasing state duration significantly would also slow down
   # the test suite, we are retrying twice on failure as a mitigation.
-  @retry(reraise=True, stop=stop_after_attempt(3))
+  @retry.with_exponential_backoff(num_retries=2, factor=1,
+                                  initial_delay_secs=0.1, max_delay_secs=0.1,
```

Comment:
Can these both be 0?

Reply:
Factor has to be 1 as we want to repeat at constant interval.
I was not sure about usage of 0 in initial delay so used 0.1 sec. As this will only be triggered in case of failure, it should not impact the test performance by much.

Reply:
Hmm... given all this and including the fact that the default retry filter is http errors, it feels like this is not really the right fit. 

Given that this is actually a test, I think simply using test dependencies make sense here, including tenacity.